envbuilder.go

```diff
@@ -932,6 +948,41 @@ func Run(ctx context.Context, options Options) error {
 
 	logf(codersdk.LogLevelInfo, "=== Running the init command %s %+v as the %q user...", options.InitCommand, initArgs, user.Username)
 
+	execOneLifecycleScript := func(s devcontainer.LifecycleScript, name string) error {
```

Comment:
Having thought about this a little more, running these after the `Setuid` isn't ideal, because they will run as `TARGET_USER`, which is the intended user for the init command, but not necessarily relevant to the lifecycle scripts.

I'm going to move these to run before the `Setuid` and use `syscall.ForkExec` so they can specify the default user for the container.

Reply:
I think that's idiomatic. You might wanna add that as a comment above that code, just so in the future it never gets adjusted!

Reply:
Done!